client/Lykke.Service.BlockchainSettings.Client/HttpClientGenerator/BlockchainSettingsControllerFactory.cs

```diff
@@ -10,30 +10,39 @@ namespace Lykke.Service.BlockchainSettings.Client.HttpClientGenerator
 {
     public class BlockchainSettingsControllerFactory : IBlockchainSettingsControllerFactory
     {
-        public IBlockchainSettingsClient CreateNew(BlockchainSettingsServiceClientSettings settings,
+        public (IBlockchainSettingsClient, IClientCacheManager) CreateNew(BlockchainSettingsServiceClientSettings settings,
             bool withCaching = true,
             params DelegatingHandler[] handlers)
         {
             return CreateNew(settings?.ServiceUrl, settings?.ApiKey, withCaching, handlers);
         }
 
-        public IBlockchainSettingsClient CreateNew(string url, string apiKey, bool withCaching = true, params DelegatingHandler[] handlers)
+        public (IBlockchainSettingsClient, IClientCacheManager) CreateNew(string url, string apiKey, bool withCaching = true, params DelegatingHandler[] handlers)
         {
+            IClientCacheManager clientCacheManager = new ClientCacheManager();
```

Comment:
Move clientCacheManager to the parameters of the method. In this case you haven't need to return a tuple. This parameter could be optional - pass DummyClientCacheManager to the builder.Create(), if null is passed